index.html

```diff
@@ -0,0 +1,1185 @@
+                            <p class="card-text">We’re used to doctors bustling in and out when in hospital, but it’s the reassuring presence and professionalism of the nursing staff that we rely on. What sets the best ones apart?</p>
+                          </div>
+                        </div>
+                        <div class="card">
+                          <img class="card-img-top" src="assets/nurse2.jpg" alt="Card image cap">
+                          <span class="sponsored">Sponsored Insight</span> 
+                          <div class="card-body">
+                            <h5 class="card-title font-weight-bold text-dark">Nurses - The Real Medical Heroes of 2020</h5>
+                            <p class="card-text">We’re used to doctors bustling in and out when in hospital, but it’s the reassuring presence and professionalism of the nursing staff that we rely on. What sets the best ones apart?</p>
+                          </div>
+                        </div>
+                        <div class="card">
+                          <img class="card-img-top" src="assets/nurse2.jpg" alt="Card image cap">
+                          <span class="sponsored">Sponsored Insight</span> 
+                          <div class="card-body">
+                            <h5 class="card-title font-weight-bold text-dark">Nurses - The Real Medical Heroes of 2020</h5>
+                            <p class="card-text">We’re used to doctors bustling in and out when in hospital, but it’s the reassuring presence and professionalism of the nursing staff that we rely on. What sets the best ones apart?</p>
+                          </div>
+                        </div>
+                        <div class="card">
+                          <img class="card-img-top" src="assets/nurse2.jpg" alt="Card image cap">
+                          <span class="sponsored">Sponsored Insight</span> 
+                          <div class="card-body">
+                            <h5 class="card-title font-weight-bold text-dark">Nurses - The Real Medical Heroes of 2020</h5>
+                            <p class="card-text">We’re used to doctors bustling in and out when in hospital, but it’s the reassuring presence and professionalism of the nursing staff that we rely on. What sets the best ones apart?</p>
+                          </div>
+                        </div>
+                     </div>                  
+                 </div>
+             </section>
+             <section class="row m-0 magazine">
+                 <div class="col-12 row p-0 m-0 magazine-top my-3">
+                    <div class="col-12 py-2 my-2 col-md-9 offset-md-3 d-flex justify-content-center align-items-center pt-md-2 pb-md-2 mb-md-3 border-custom">
+                         <span class="icon">N</span>
+                         <h3 class="m-0">In the magazine</h3>
+                    </div>  
```

Comment:
Line number 434: content "In the magazine" should be uppercase, kindly recommended to fix it